src/util/rocksdb.rs

```diff
@@ -205,31 +214,31 @@ impl SliceTransform for FixedPrefixSliceTransform {
 mod tests {
     use rocksdb::{DB, Options};
     use tempdir::TempDir;
-    use util::collections::HashMap;
-    use super::check_and_open;
+    use super::{check_and_open, CFOptions};
 
     #[test]
     fn test_check_and_open() {
         let path = TempDir::new("_util_rocksdb_test_check_column_families").expect("");
         let path_str = path.path().to_str().unwrap();
 
         // create db when db not exist
-        let cfs_opts = map!["default" => Options::new()];
+        let cfs_opts = vec![CFOptions::new("default", Options::new())];
```

Comment:
There is a var `CF_DEFAULT` defined to be "default", which is used everywhere. We could use that.